stdlib/3/distutils/dist.pyi

```diff
@@ -1,8 +1,56 @@
 from distutils.cmd import Command
-from typing import Any, Dict, Iterable, Mapping, Optional, Tuple, Type
+from typing import IO, Any, AnyStr, Dict, Iterable, List, Mapping, Optional, Tuple, Type
+
+class DistributionMetadata:
+    def __init__(self, path: Optional[str] = ...): ...
+    name: Optional[str]
+    version: Optional[str]
+    author: Optional[str]
+    author_email: Optional[str]
+    maintainer: Optional[str]
+    maintainer_email: Optional[str]
+    url: Optional[str]
+    license: Optional[str]
+    description: Optional[str]
+    long_description: Optional[str]
+    keywords: Optional[str]
+    platforms: Optional[str]
+    classifiers: Optional[str]
+    download_url: Optional[str]
+    provides: Optional[str]
+    requires: Optional[str]
+    obsoletes: Optional[str]
+    def read_pkg_file(self, file: IO[AnyStr]) -> None: ...
+    def write_pkg_info(self, base_dir: str) -> None: ...
+    def write_pkg_file(self, file: IO[AnyStr]) -> None: ...
+    def get_name(self) -> str: ...
+    def get_version(self) -> str: ...
+    def get_fullname(self) -> str: ...
+    def get_author(self) -> str: ...
+    def get_author_email(self) -> str: ...
+    def get_maintainer(self) -> str: ...
+    def get_maintainer_email(self) -> str: ...
+    def get_contact(self) -> str: ...
+    def get_contact_email(self) -> str: ...
+    def get_url(self) -> str: ...
+    def get_license(self) -> str: ...
+    def get_licence(self) -> str: ...
+    def get_description(self) -> str: ...
+    def get_long_description(self) -> str: ...
+    def get_keywords(self) -> List[str]: ...
+    def get_platforms(self) -> List[str]: ...
+    def get_classifiers(self) -> List[str]: ...
```

Comment:
These three can also return a `str` under certain circumstances. Usually we recommend not to return unions, but in this case I'd recommend to use `Union[str, List[str]]` as return type.